tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,32 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _name_locator = (By.CSS_SELECTOR, ".item > .info > h3")
```

Comment:
The name locator should be within the `Collection` region. This should be an element that represents the collection at a higher level than the name. For example this could be `.item` and then `_name_locator` could be `.info > h3`.